packages/eslint-plugin/tools/generate-configs.ts

```diff
@@ -76,10 +76,7 @@ async function main(): Promise<void> {
   );
   const EXTENDS = ['./configs/base', './configs/eslint-recommended'];
 
-  type RuleEntry = [
-    string,
-    TSESLint.RuleModule<string, readonly unknown[], TSESLint.RuleListener>,
-  ];
+  type RuleEntry = [string, TSESLint.RuleModule<string, readonly unknown[]>];
```

Comment:
TODO - should really migrate to `import { RuleModule } from '@typescript-eslint/utils/ts-eslint'`
That was the entire reason for adding the export maps